database/seeders/OtherDataSeeder.php

```diff
@@ -160,4 +173,73 @@ public function run()
         $seeder_end = microtime(true);
         echo "\e[32mSeeded:\e[0m    \e[1mOtherDataSeeder\e[0m (".round(($seeder_end - $seeder_start), 2).'s)'.PHP_EOL;
     }
+
+    // Withdrawal seeding helper method, based on store() method in WithdrawalController.php
+    // NOTE: Any updates in that method should be mirrorred in here
+    public function seedWithdrawals(int $index, int $maxMoney, int $maxOrderlinesInWithdrawal): int
```

Comment:
I think it should be possible to just use the WithdrawalController::store() method if you create a request object. That would avoid us having to mirror any changes between this function and the controller. https://stackoverflow.com/questions/40262947/create-a-laravel-request-object-on-the-fly